models/product-variant/src/generator.ts

```diff
@@ -0,0 +1,26 @@
+import * as Attribute from '@commercetools-test-data/attribute';
+import { fake, Generator } from '@commercetools-test-data/core';
+import * as Image from '@commercetools-test-data/image';
+import * as Price from '@commercetools-test-data/price';
+import { TProductVariant } from './types';
+
+// https://docs.commercetools.com/api/projects/products#productvariant
+
+const generator = Generator<TProductVariant>({
+  fields: {
+    id: fake((f) => f.datatype.uuid()),
+    key: fake((f) => f.lorem.slug(2)),
+    sku: fake((f) => f.random.word()),
+    prices: fake(() => [Price.random()]),
+    price: null,
+    attributes: fake(() => [Attribute.random()]),
+    images: fake(() => [Image.random()]),
+    assets: [],
+    availability: null,
+    isMatchingVariant: fake((f) => f.datatype.boolean()),
```

Comment:
I would keep it as `false` by default. This is only used for product projection search results.

<img width="807" alt="image" src="https://user-images.githubusercontent.com/1110551/221662564-b5e6a853-7e75-434b-8ba7-8ee57b95a770.png">